README.md

```diff
@@ -1,35 +1,47 @@
 # visualize_packs
+Visualize_packs helps you visualize the structure, intended and actual, of your package-based Ruby application.
 
-This gem helps visualize relationships between packs.
+This gem takes a minimal approach in that it only outputs a graph in the format of [graphviz](https://graphviz.org/)' [dot language](https://graphviz.org/doc/info/lang.html). Install graphviz and use one of the chains of commands from below to generate full or partial images of the graphs of your application.
 
-![Example of visualization](docs/example.png)
+## Visualize your entire application
+```
+bundle exec visualize_packs > packs.dot && dot packs.dot -Tpng -o packs.png && open packs.png
+```
+
+## packs.png for every package in your app
+
+This will generate a local dependency diagram for every pack in your app
 
-# CLI Usage
-## bin/packs
-For simpler use, add `bin/packs` via `use_packs` (https://github.com/rubyatscale/use_packs)
 ```
-bin/packs visualize # all packs
-bin/packs visualize packs/a packs/b # subset of packs
-bin/packs # enter interactive mode to select what packs to visualize
+find . -iname 'package.yml' | sed 's/\/package.yml//g' | sed 's/\.\///' | xargs -I % sh -c "bundle exec visualize_packs --only=% > %/packs.dot && dot %/packs.dot -Tpng -o %/packs.png"
 ```
 
-# Ruby API Usage
-## Building a package graph for a selection of packages
-```ruby
-# Select the packs you want to include
-selected_packs = Packs.all
-selected_packs = Packs.all.select{ |p| ['packs/my_pack_1', 'packs/my_pack_2'].include?(p.name) }
-selected_packs = Packs.all.select{ |p| ['Team 1', 'Team 2'].include?(CodeOwnership.for_package(p)&.name) }
-VisualizePacks.package_graph!(selected_packs)
+If your app is large and has many packages and violations, the above graphs will likely be too big. Try this version to get only the edges to and from the focus package for each diagram:
+
 ```
+find . -iname 'package.yml' | sed 's/\/package.yml//g' | sed 's/\.\///' | xargs -I % sh -c "bundle exec visualize_packs --only=% --only-edges-to-focus > %/packs.dot && dot %/packs.dot -Tpng -o %/packs.png"
```

Comment:
I wonder if there's a way to make this part of the CLI API. I think bash stuff is a bit intimidating for some folks. Not a big deal since this works for now – just trying to find ways the user-facing docs can present simpler APIs.

Reply:
Let's see if this becomes something that we get feedback everyone does... Otherwise it adds a bunch of complexity inside the gem